hack/flake-report-creator.sh

```diff
@@ -1,12 +1,13 @@
 #!/usr/bin/bash
 
 tmp_dir="$(mktemp -d)"
-docker run -v /etc/pki:/etc/pki -v /etc/ssl:/etc/ssl \
+docker run \
         -v "$(dirname $(echo $GOOGLE_APPLICATION_CREDENTIALS)):$(dirname $(echo $GOOGLE_APPLICATION_CREDENTIALS))" \
         -e GOOGLE_APPLICATION_CREDENTIALS="$GOOGLE_APPLICATION_CREDENTIALS" \
         -v "${tmp_dir}:/tmp:Z" \
         --network host \
-        quay.io/kubevirtci/flake-report-creator:v20220208-ad538bce \
+        quay.io/kubevirtci/flake-report-creator:v20220208-9141ecb9 \
+        --insecureSkipVerify \
```

Comment:
what exactly are we querying which uses untrusted certificates?

Reply:
@rmohr actually the problem is IIUC that the image doesn't have the root ca certificates to verify the server certificate and then fails.

```
docker run --network host quay.io/kubevirtci/flake-report-creator:v20220208-9141ecb9 jenkins                                                                                                                   11s
{"level":"info","msg":"Creating client for https://main-jenkins-csb-cnvqe.apps.ocp-c1.prod.psi.redhat.com/","time":"2022-02-09T12:02:47Z","type":"jenkins"}
Error: failed to contact jenkins https://main-jenkins-csb-cnvqe.apps.ocp-c1.prod.psi.redhat.com/: Get "https://main-jenkins-csb-cnvqe.apps.ocp-c1.prod.psi.redhat.com/api/json": x509: certificate signed by unknown authority
```

I worked around this by mounting the `/etc/pki` and `/etc/ssl` directories in the script. I guess we could just bake them into the image. Or would there be a better approach?

Reply:
Hm, I wonder what certificate that is. There are ca certs included in `distroless` images. However it may be that not enough are: https://github.com/GoogleContainerTools/distroless/issues/753

Reply:
If we want to keep the flag for this specific service, just consider moving it to the caller of the script and pass it with the other "$@" commands.